subsys/bluetooth/controller/ll_sw/ull_scan_types.h

```diff
@@ -43,7 +43,7 @@ struct ll_scan_aux_set {
 	struct lll_scan_aux lll;
 
 	/* lll_scan or lll_sync */
-	void *parent;
+	void *volatile parent;
```

Comment:
Should this be `volatile void *parent`? I see this format used in another couple of places but it looks unusual to me, the vast majority of "volatile" uses in the code base use that first I think, and the pointer next to the variable name. Up to you really, I guess we have no standard, it just looks unusual to me.

Reply:
There is a difference in `volatile void *parent;` (or `void volatile *parent;`) here, `parent` points to an address whose content is volatile. Where as `void *volatile parent` here, content of `parent` is volatile (like  `parent` was modified in an ISR).

Reply:
Ohh I see, makes sense, thanks for explaining.